hw_diag/templates/diagnostics_page.html

```diff
@@ -41,8 +47,13 @@ <h3 class="text-center mb-4">Diagnostics Breakdown</h3>
             <div class="card mb-0 h-100">
               <table class="table dt-responsive nowrap m-2 w-auto">
                 <tr>
-                  <td class="border-0">Helium Address</td>
-                  <td class="border-0 text-right"><a href="https://explorer.helium.com/hotspots/{{ diagnostics.PK }}" target="_blank">View on Explorer</a></td>
+                  {% if diagnositcs.PK == "ECC failure" %}
+                    <td class="border-0">Helium Address</td>
+                    <td class="border-0 text-right"><a href="https://explorer.helium.com/hotspots/{{ diagnostics.PK }}" target="_blank">View on Explorer</a></td>
```

Comment:
Think this is the wrong way around

Reply:
Thank you for bringing this up, I changed it on the device via SSH but forgot to change it in the code after testing